recipe/meta.yaml

```
@@ -24,14 +24,18 @@ requirements:
     - python
     - pytz
     - billiard >=3.6.3,<4.0
-    - kombu >=4.6.8,<4.7
+    - future >=0.18.0
+    - kombu >=4.6.10,<4.7
+    - vine 1.3.0.*
     - sqlalchemy
     - cryptography
 
```

We may also want to add
```suggestion
  run_constrained:
    - sphinx >=2.0

```
due to https://github.com/celery/celery/pull/6150/files#diff-d16507d7290ca403dd8c7a3c4b1df02fR10.
@conda-forge/celery, can/should I add that or is there any reason not to?